records/new/nodejs-learn-package-publish.json

```
@@ -0,0 +1,5 @@
+{
+  "action": "post",
+  "account": "NODEJS_ORG",
+  "richText": "Trying to publish a package and there are just. so. many. options? The good news is it's probably not as complicated as it seems. We gotchu. \nCheck out the new https://nodejs.org/en/learn/modules/publishing-a-package"
```

I think 2 line breaks so there's an empty line between? Otherwise, no line breaks?

```suggestion
  "richText": "Trying to publish a package and there are just. so. many. options? The good news is it's probably not as complicated as it seems. We gotchu.\n\nCheck out the new https://nodejs.org/en/learn/modules/publishing-a-package"
```

---

Double return ?

---

Yeah

> Trying to publish a package and there are just. so. many. options? The good news is it's probably not as complicated as it seems. We gotchu.
>
> Check out the new https://nodejs.org/en/learn/modules/publishing-a-package

Vs

> Trying to publish a package and there are just. so. many. options? The good news is it's probably not as complicated as it seems. We gotchu.
> Check out the new https://nodejs.org/en/learn/modules/publishing-a-package

Vs

> Trying to publish a package and there are just. so. many. options? The good news is it's probably not as complicated as it seems. We gotchu. Check out the new https://nodejs.org/en/learn/modules/publishing-a-package